### PR TITLE
docs: Minor formatting & punctuation tweaks to native-image.md

### PR DESCRIPTION
**Pull request by Zearin**
#### Description of Change

Minor formatting and punctuation tweaks.

#### Checklist

- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/api/native-image.md [hunk 1]
```diff
@@ -273,15 +273,15 @@ Returns `NativeImage` - The cropped image.
   * `width` Integer (optional) - Defaults to the image's width.
   * `height` Integer (optional) - Defaults to the image's height.
   * `quality` String (optional) - The desired quality of the resize image.
-    Possible values are `good`, `better` or `best`. The default is `best`.
+    Possible values are `good`, `better`, or `best`. The default is `best`.
```
Comment (MarshallOfSound) [inline docs/api/native-image.md]: We can make this `String` be typed as an enum if we write "Can be X, Y or Z"

```suggestion
    Can be `good`, `better` or `best`. The default is `best`.
```

Comment (codebytere) [reply docs/api/native-image.md]: @Zearin i'll approve once the above comment has been addressed ☝️

**Comment by miniak** [comment]
@Zearin do you still want to get this PR merged?

**Comment by zcbenz** [comment]
I'm closing this PR since there has been no update from the op for 5 months.

**Pull request closed without merging**